tests/factories/pages.py

```diff
@@ -1,11 +1,14 @@
+import datetime
+
 import factory
+from factory.fuzzy import FuzzyDate
```

Comment:
factory.fuzzy is deprecated according to the docs: https://factoryboy.readthedocs.io/en/latest/fuzzy.html